nss-day-cohort-51/nss-day-cohort-51.github.io
# Ss update modal 89

# Description - updated modal

## Type of change


# Testing Instructions
Fetch branch, serve, and click on modal to see if content shows up including tech buttons

# Checklist:
